<?xml version="1.0"?>
<launch>
<param name="use_sim_time" value="true" />
<group ns="robot_0">
<node pkg="nav2d_karto" name="Mapper" type="mapper">
<!--    <remap from="karto_in" to="/roscbt/robot_0/karto_in" />-->
    <remap from="karto_out" to="/karto_out" />
</node>
</group>
<group ns="robot_1">
<node pkg="nav2d_karto" name="Mapper" type="mapper">
<!--    <remap from="karto_in" to="/roscbt/robot_1/karto_in" />-->
    <remap from="karto_out" to="/karto_out" />
</node>
</group>
<!--    <group ns="robot_2">-->
<!--    <node pkg="nav2d_karto" name="Mapper" type="mapper">-->
<!--&lt;!&ndash;    <remap from="karto_in" to="/roscbt/robot_2/karto_in"/>&ndash;&gt;-->
<!--    <remap from="karto_out" to="/karto_out" />-->
<!--</node>-->
<!--    </group>-->

<!--    <group ns="robot_3">-->
<!--    <node pkg="nav2d_karto" name="Mapper" type="mapper">-->
<!--&lt;!&ndash;    <remap from="karto_in" to="/roscbt/robot_3/karto_in" />&ndash;&gt;-->
<!--    <remap from="karto_out" to="/karto_out" />-->
<!--</node>-->
<!--    </group>-->

<!--        <group ns="robot_4">-->
<!--    <node pkg="nav2d_karto" name="Mapper" type="mapper">-->
<!--&lt;!&ndash;    <remap from="karto_in" to="/roscbt/robot_4/karto_in" />&ndash;&gt;-->
<!--    <remap from="karto_out" to="/karto_out" />-->
<!--</node>-->
<!--    </group>-->


    <node name="roscbt" pkg="gvgexploration" type="roscbt.py" respawn="true">
        <rosparam file="$(find gvgexploration)/param/roscbt_config.yaml"/>
    </node>

</launch>
